### ISSUE TITLE
Unhandled Request Responses

### ISSUE THREAD

**Issue by UnrecoverableFault** [slackapi/bolt-python#299]
If I have a event that is not acknowledged by my application, I will get a warning of a unhandled request. I would just like to ack() all requests that do not apply to my application so I'm not sending 404's back to Slack when it's really fine to send 200. The event is im.history so it can very loud with various unmatched message types. 

#### The `slack_bolt` version

```
slack-bolt==1.4.4
slack-sdk==3.4.2
```

#### Python runtime version

`Python 3.9.2`

#### OS info

`ProductName:    macOS
ProductVersion: 11.2.3
BuildVersion:   20D91
Darwin Kernel Version 20.3.0: Thu Jan 21 00:07:06 PST 2021; root:xnu-7195.81.3~1/RELEASE_X86_64`

#### Steps to reproduce:

1. Handle requests I want to handle

### Expected result:

Does not WARN to logs, instead ignores with 200

### Actual result:

WARN's to logs

## Requirements

Please read the [Contributing guidelines](https://github.com/slackapi/bolt-python/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to those rules.

**Comment by seratch** [comment]
Hi @UnrecoverableFault, thanks for taking the time to write in!

We just released a new minor version v1.5.0 yesterday and the version includes a solution for your use case: https://github.com/slackapi/bolt-python/releases/tag/v1.5.0

>Does not WARN to logs, instead ignores with 200

The following code should work as you expect.

```python
import os
from slack_bolt import App

app = App(
    token=os.environ["SLACK_BOT_TOKEN"],
    signing_secret=os.environ["SLACK_SIGNING_SECRET"],
    # enable @app.error handler to catch the patterns
    raise_error_for_unhandled_request=True,
)

@app.error
def handle_errors(error):
    if isinstance(error, BoltUnhandledRequestError):
        # you may want to have some logging here
        return BoltResponse(status=200, body="")
    else:
        # other error patterns
        return BoltResponse(status=500, body="Something Wrong")
```

I hope this was helpful to you. If everything is clear now, may I ask you to close this issue?

**Comment by UnrecoverableFault** [comment]
Absolutely! I'll go ahead and close it.
